service/lib/agama/software/manager.rb

```diff
@@ -443,6 +447,14 @@ def locale=(locale)
       # @return [Logger]
       attr_reader :logger
 
+      # Decides whether grub-bls should be used for given product
+      def use_bls?(id)
+        return true if id =~ /Tumbleweed/
+        return true if id =~ /Slowroll/
```

Comment:
Cool. But I would make this a setting at the Agama product file (you know, those under `product.d`), instead of hardcoding product names in the code.

Reply:
sure ...  this is in proof of concept only now.

no problem when we're on same continent this time it is not  problem to move between countries ;-)